apps/condo/domains/common/components/PostMessageProvider/validators.ts

```diff
@@ -16,7 +16,7 @@ const NoParamsValidator = ajv.compile(NoParamsSchema)
 const CondoWebSendAnalyticsEventParamsSchema = {
     type: 'object',
     properties: {
-        event: { type: 'string', enum: ['click', 'check'] },
+        event: { type: 'string', enum: ['click', 'check', 'change', 'select'] },
```

Comment:
How about removing select to change btw if you're touching analytics btw?)

Reply:
Done that